docs/concepts/usage-ideas-examples.md

```diff
@@ -2,14 +2,317 @@
+
+Sensor data is typically-structured and often contains duplicated blocks of data. In cases where a lot of sensor data is recorded, IPFS can reduce disk usage even more than simple compression [thanks to its de-duplication techniques](http://blog.klaehn.org/2018/06/10/efficient-telemetry-storage-on-ipfs/).
+
+## Decentralize your data
+
+By decentralizing your data, you increase availability in case your server runs into an issue, your ISP is unhappy with what you publish, or a hostile government issues a take-down-order. You will also decrease the load time for your swarm-connected users, and natively enable your app to work offline or on local networks. Depending on your app, your users can also reduce their storage requirements and download time thanks to native de-duplication and caching.
+
+### Decentralized database
+
+[OrbitDB](https://github.com/orbitdb/orbit-db) is a serverless, distributed, peer-to-peer database. OrbitDB uses IPFS as its data storage and IPFS Pubsub to automatically sync databases with peers. It's an eventually consistent database that uses CRDTs for conflict-free database merges, making OrbitDB an excellent choice for decentralized apps (dApps), blockchain applications, and offline-first web applications. There are Go and Javascript implementations available.
+
+If you are currently using MongoDB, you might be more comfortable with [ThreadDB](https://docs.textile.io/threads/introduction/) or [AvionDB](https://github.com/dappkit/aviondb-onboard), which are unstructured databases built on top of IPFS.
+
+### IPFS hosting with Textile
+
+[Textile](https://Textile.io) is a hosting company on IPFS developing the layers on top of IPFS. Among other things, they came up with separated _cloud_ environments called [buckets](https://docs.textile.io/hub/buckets/). Textile has also built an impressive toolset for [building decentralized applications and integrations](https://blog.textile.io/announcing-the-textile-protocol-hub/).
+
+### AWS S3 integration
+
+If you are using Amazon Web Service's S3 connections for your cloud, you can use [RTrade's storage server](https://github.com/RTradeLtd/s3x) to replicate it on IPFS with minimal disruption to your workflow. It is forked from the popular object storage server [MinIO](https://min.io/) and will be re-integrated into MinIO natively sometime soon.
+
+### NextCloud integration
+
+JusticeNode has created [an extention](https://github.com/justicenode/files_external_ipfs) for the popular self-hosted cloud service NextCloud. This integration allows users to use IPFS as external storage.
+
+### Deploy your GitHub website on IPFS
+
+[Fleek.co](https://fleek.co/) allows you to effortlessly build sites & apps on IPFS. The workflow is similar to that of Netlify: developers can link their GitHub project to Fleek and have it automatically built and deployed on IPFS whenever a change is made to a specific branch. Fleek also comes with ENS and domain name integration.
```

Comment:
People from Fleek says they will support deploying several more methods of deploying website (https://discuss.ipfs.io/t/publish-website-on-ipfs/7957/5) apart from Github, and I believe them (they are moving fast). I don't know if it's worth mentioning or not.

Reply:
In that case it's probably worth mentioning what Fleek are intending to support in the future.

Reply:
Done